apps/chrome-devtools/src/extension/background.ts

```diff
@@ -72,33 +86,35 @@ const applyActivateState = async (appName: string, tabId: number) => {
     await chrome.scripting.executeScript({
       target: { tabId },
       args: [dataType, JSON.stringify(content), postMessageType],
-      // eslint-disable-next-line prefer-arrow/prefer-arrow-functions, @typescript-eslint/no-shadow
-      func: function (dataType, content, postMessageType) {
+      // eslint-disable-next-line prefer-arrow/prefer-arrow-functions -- expecting fat function to handle the context correctly
```

Comment:
unless I'm missing something, we are not actually using the context (`this`) in this function

Reply:
Nor we use an arrow function?